api/DTOs/SupplierDTO.cs

```diff
@@ -0,0 +1,28 @@
+using Microsoft.AspNetCore.Mvc;
+
+namespace DTO.Supplier;
+
+[ApiExplorerSettings(IgnoreApi = true)]
+public class SupplierRequest : BaseDTO
+{
+    public string? Code { get; set; }
+    public string? Name { get; set; }
+    public string? Reference { get; set; }
+    public Guid? Contact_id { get; set; }
+    public ContactDTO? Contact { get; set; }
+    public Guid? Address_id { get; set; }
```

Comment:
Graag PascalCase aanhouden
```suggestion
    public Guid? AddressId{ get; set; }
```